framework/src/media/audio/audio_manager.c

```diff
@@ -217,54 +217,70 @@ audio_manager_result_t find_audio_card(audio_card_type_t card_type)
 	return ret;
 }
 
+static audio_manager_result_t find_audio_card_if_empty(audio_card_type_t card_type)
+{
+	int id = g_actual_audio_out_card_id;
+
+	if (card_type == INPUT) {
+		id = g_actual_audio_in_card_id;
+	}
+	
+	if ((id == INVALID_ID) && (find_audio_card(card_type) != AUDIO_MANAGER_SUCCESS)) {
```

Comment:
@Taejun-Kwon is that correct condition~?
i think change to or codition

Reply:
yes it is correct. if id is INVALID_ID, it means there's no founded card.
Hence find_audio_card called.